package.json

```diff
@@ -27,9 +27,10 @@
     "test:watch": "jest --watch",
     "test:debug": "node --debug-brk --inspect ./node_modules/.bin/jest -i",
     "test:karma": "karma start --single-run",
-    "test:travis": "npm run lint && npm run test",
-    "lint": "node_modules/.bin/eslint .",
-    "style": "npm run lint"
+    "test:travis": "npm run lint && npm run test:karma",
+    "lint": "eslint .",
```

Comment:
Depending on how your PATH is set, this may execute using a global eslint that is not necessarily the same version as what's specified for this repo. It's not a huge deal but it has given me some annoying problems in the past, which is why I wrote out the relative path.